toonz/sources/toonz/exportxsheetpdf.cpp

```diff
@@ -1549,8 +1549,8 @@ void XSheetPDFTemplate::initializePage(QPdfWriter& writer) {
   QPageLayout pageLayout;
   pageLayout.setUnits(QPageLayout::Millimeter);
   pageLayout.setPageSize(
-      QPageSize(m_p.documentPageSize));  // ���ʂ�B4��ISO B4(250x353mm)
-                                         // ��{��B4��JIS B4(257x364mm)
+      QPageSize(m_p.documentPageSize));  // ���ʂ�B4��ISO B4(250x353mm)
+                                         // ���{��B4��JIS B4(257x364mm)
```

Comment:
Sorry about this! I am so puzzled how these edits are making it in to my diffs. I guess it's something to do with Gitkraken, the gui git client I'm using. :angry: I'll try to revert this in vim